### PR TITLE
Update rtmps-server.xml

### PR DESCRIPTION
**Pull request by JamiePhonic**
Tweak formatting for readability

### CODE REVIEW

**Review by Roxedus — CHANGES_REQUESTED** [review]
These are illegal characters in CA afaik

**Comment by JamiePhonic** [comment]
changes made, should be compliant now

File: templates/rtmps-server.xml [hunk 1]
```diff
@@ -11,46 +19,46 @@
...
+http://(server ip):8080/hls/(stream-key).m3u8 &#xD;
+http://(server ip):8080/dash/(stream-key)_src.mpd&#xD;
+&#xD;
+View in Browser (assumes (stream-key) is 'test', edit the html files from the mounted directory to change this):&#xD;
+http://(server ip):8080/players/hls.html&#xD;
+http://(server ip):8080/players/dash.html&#xD;
+http://(server ip):8080/players/rtmp.html</Overview>
   <Category>GameServers: Network:Other</Category>
-  <WebUI>http://[IP]:[PORT:8080]/stats</WebUI>
+  <WebUI>http://(IP):(PORT:8080)/stats</WebUI>
```
Comment (Roxedus) [inline templates/rtmps-server.xml:40]: These need to be square brackets

Comment (JamiePhonic) [reply templates/rtmps-server.xml]: Thats whatb CTRL+H gets me...
Fixed now.

**Review by Roxedus — APPROVED** [review]

**Pull request merged**